src/utils/string/StringUtils.js

```diff
@@ -14,7 +14,7 @@ import {getAnonymousUserId} from '../users';
 import {getUserId} from '../token';
 
 const NO_POLL_UUID = '00000000-0000-0000-0000-000000000000';
-
+const urlRegex = /(https?:\/\/\S+)+/g;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `urlRegex` regular expression only checks if the string starts with "http://" or "https://", and contains at least one non-space character after that. This might not be sufficient to validate all URLs. Consider using a more comprehensive regex for URL validation.
```diff
- const urlRegex = /(https?:\/\/\S+)+/g;
+ const urlRegex = /^(https?|chrome):\/\/[^\s$.?#].[^\s]*$/gm;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->